hdrop: 0.3.0 -> 0.4.0 (#85)
hyprwm/contrib: hdrop/README.md

#### Options:

> -b, --background  
> Changes the default behaviour: if the specified program is not running, launch it in the background instead of in the foreground. Thereafter `hdrop -b` will work the same as without this flag.
>
> -c, --class  

> > Set classname of the program to be run. Use this if the classname is different from the name of the [COMMAND] and hdrop has not yet a hardcoded replacement.

Reviewer: ```suggestion
> Set classname of the program to be run. Use this if the classname is different from the name of the [COMMAND] and hdrop does not have a hardcoded replacement.
```

If hdrop can't match an already running program and starts a new instance instead, then its class name is most likely different from its command name. For example, the class name of `telegram-desktop` is `org.telegram.desktop` and the class name of `logseq` is `Logseq`.

Run `hdrop -v [COMMAND]` _in the terminal_ to see maximum output for troubleshooting and find out the actual class name. Then use `hdrop -c CLASSNAME` to make it work. `hdrop -i [COMMAND]` might be sufficient, as long as a case insensitive (partial) match is sufficient.

> Please report instances of programs with differing class names, so that i can add them to `hdrop`.

Reviewer: ```suggestion
Please report instances of programs with differing class names, so that they can be added to `hdrop`.
```
